lib/adaptors/serialport.js

```diff
@@ -52,13 +52,13 @@ util.inherits(Adaptor, EventEmitter);
  */
 Adaptor.prototype.open = function open(callback) {
   var self = this,
-      port = this.serialport = new serialport.SerialPort(this.conn, {});
+      port = this.serialport = new serialport.SerialPort(this.conn, {}, false);
 
   function emit(name) {
     return self.emit.bind(self, name);
   }
 
-  port.on("open", function(error) {
+  port.open(function(error) {
```

Comment:
The reason for this change is described in #30. I'm not sure why the test fails, the code has been successfully working for me since the change.